api/config/config_staging.json

```diff
@@ -1,19 +1,24 @@
 {
   "firecloud": {
     "baseUrl": "https:\/\/api.firecloud.org",
-    "billingAccountId": "014D91-FCB792-33D2C0",
-    "billingProjectPrefix": "aou-rw-staging-",
     "clusterMaxAgeDays": 7,
     "clusterIdleMaxAgeDays": 3,
     "debugEndpoints": false,
     "enforceRegistered": true,
     "leoBaseUrl": "https:\/\/notebooks.firecloud.org",
     "registeredDomainName": "all-of-us-registered-staging",
-    "billingRetryCount": 2,
-    "billingProjectBufferCapacity": 10,
     "xAppIdValue": "staging-AoU-RW",
     "vpcServicePerimeterName": ""
   },
+  "billing": {
+    "billingAccountId": "014D91-FCB792-33D2C0",
```

Comment:
since these are all scoped under billing, do we need the 'billing' prefix as well?